codility_training/lessons.lesson09.MaximumSliceProblem.MaxDoubleSliceSum/sAp00n.py

```diff
@@ -0,0 +1,43 @@
+# test case [5, 17, 0, 3], [0, 10, -5, -2, 0], [5, 5, 5] and all negative
+
+def solution(A):
+    if len(A) <= 3:
+        return 0
+    else:
+        B = A[1:-1]
+        print(f'B = {B}')
+        start_ind, end_ind = find_max_slice(B)
+
+        resulte = B[start_ind:end_ind]
+        if end_ind == None:
+            print(f'B[{start_ind}:{end_ind}] =  [{A[start_ind]} {resulte} {A[-1]}]')
+        else:
+            print(f'B[{start_ind}:{end_ind}] =  [{A[start_ind]} {resulte} {A[end_ind + 1]}]')
+        if len(resulte) == 1:
+            if resulte[0] > 0:
+                return resulte[0]
+            else:
+                return 0
+        else:
+            resulte.remove(min(resulte))
+            return sum(resulte)
```

Comment:
```suggestion
        if len(resulte) == 1:
            return resulte[0] if resulte[0] > 0 else 0
        resulte.remove(min(resulte))
        return sum(resulte)
```
불필요한 if ~ else는 사용하지 않는게 좋습니다. 삼항연산자를 잘 이용해주세요!